css/home.css

```diff
@@ -0,0 +1,90 @@
+nav{
+    width: 100vw;
+    padding: 0 5vw;
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+    position: fixed;
+    font-size: 1.2rem;
+    background: linear-gradient(180deg, #1cb5ee, #199FDA);
+}
+
+.nav-list{
+    display: flex;
+}
+
+.nav-list li{
+    padding: 1vh 0;
+    list-style: none;
+    display: flex;
+}
+
+.nav-list a {
+    padding: 1vh 1vw;
+    color: black;
+    text-decoration: none;
+    transition: background .5s ease;
+}
+
+.nav-list a:hover {
+    background: white;
+}
```

Comment:
change the text color to orange or something rather than changing the background color.